tests/test_legal.py

```diff
@@ -210,7 +210,12 @@ def test_query_dsl_with_ao_category_filter(self, es_search):
                 'bool': {
                     'must': [
                         {'term': {'_type': 'advisory_opinions'}},
-                        {'match': {'_all': 'president'}},
+                        {'bool': {'minimum_should_match': 1}}
```

Comment:
I don't understand this line, isn't the query covered by the minimum_should_match from below?